lib/src/config_handler.dart

```diff
@@ -50,8 +52,8 @@ class ConfigHandler {
     required this.initializer,
   }) : configFile = fs.file(p.join(
           homeDirectory.path,
-          '.dart-tool',
-          'dart-flutter-telemetry.config',
+          kDartToolDirectoryName,
```

Comment:
Should this be `'.dart'`?

Reply:
I'm not too opinionated on this. I just suggested `.dart` because I already have a user-home level `.dart` but not `.dart-tool`, and I thought `.dart-tool` was specifically for projects.

Reply:
That is a good point, I will relabel this constant